test/e2e/tests/confirmations/transactions/metrics.spec.ts

```diff
@@ -79,6 +86,8 @@ describe('Metrics', function () {
 
         const events = await getEventPayloads(driver, mockedEndpoints);
 
+        // This is left for debugging purposes
+        console.log(events);
```

Comment:
this is intentionally left. Before, for debugging an error in this test, we only saw

`AssertionError [ERR_ASSERTION]: Expected values to be strictly equal: + actual - expected  + true - undefined      `

This is not very helpful. To debug this properly we need to see all the events that happened and in which order to see what is exactly failing (either a change in order, a specific event property etc)